mobile/lib/blocs/insights/insights_bloc.dart

```diff
@@ -42,7 +42,9 @@ class InsightsBloc extends Bloc<InsightsEvent, InsightsState> {
         forecastAirQuality: todayForecast.airQuality,
         forecastPm2_5: todayForecast.pm2_5,
       );
-    } catch (_) {}
+    } catch (exception, stackTrace) {
+      logException(exception, stackTrace);
```

Comment:
@jordanopos do we need to "await" here too?

Reply:
Hey, @Baalmart , same thing here.

Reply:
Alright @jordanopos